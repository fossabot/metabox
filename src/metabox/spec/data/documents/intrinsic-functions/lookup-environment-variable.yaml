Metabox:
  Description: Lookup environment variables
  TopValueName: "Fn::Env TopValue"

  Parameters:
    TopParameterName: "Fn::Env TopParameterValue"

  Resources:
    Paramerters:
      ResourceParameterName: "Fn::Env ResourceParameterName"

    Resource1:
      Properties:
        PropertyValueFunction: "Fn::Env PropertyValueFunction"

    Resource2:
      Properties:
        PropertyValueToken: "${ENV:PropertyValueToken}"
        "${ENV:PropertyNameToken}": "PropertyNameToken"
        "${ENV:PropertyNameToken}": "${ENV:PropertyValueToken}"

    Resource3:
      Properties:
        "Prefix-${ENV:PropertyNameToken}": "Value1"
        "${ENV:PropertyNameToken}-Postfix": "Value2"
        "Prefix-${ENV:PropertyNameToken}-Postfix": "Value3"