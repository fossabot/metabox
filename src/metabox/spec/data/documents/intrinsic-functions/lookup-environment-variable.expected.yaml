Metabox:
  Description: Lookup environment variables
  TopValueName: TopValue1

  Parameters:
    TopParameterName: TopParameterValue1

  Resources:
    Paramerters:
      ResourceParameterName: ResourceParameterName1

    Resource1:
      Properties:
        PropertyValueFunction: PropertyValueFunction1

    Resource2:
      Properties:
        PropertyValueToken: PropertyValueToken1
        PropertyNameToken1: "PropertyNameToken"
        PropertyNameToken1: PropertyValueToken1

    Resource3:
      Properties:
        "Prefix-PropertyNameToken1": "Value1"
        "PropertyNameToken1-Postfix": "Value2"
        "Prefix-PropertyNameToken1-Postfix": "Value3"